mountpoint-s3/scripts/fs_bench.sh

```diff
@@ -104,8 +108,20 @@ should_run_job() {
     fi
 }
 
-read_benchmark () {
-  jobs_dir=mountpoint-s3/scripts/fio/read
+# Run all benchmarks within a category.  Fio job definitions should exist under a directory
+# with the cateogry name, passed as first argument, inside mountpoint-s3/scripts/fio.
+#
+# Params:
+# $1: benchmark category.
+run_benchmarks() {
+  category=$1
+  jobs_dir=mountpoint-s3/scripts/fio/$category
+
+  if [ $category == "read" ]; then
+    part_size="--part-size=16777216"
+  else
+    part_size=""
+  fi
```

Comment:
I want to suggest that we use `--write-part-size`, to only impact the files uploaded.

This way, the read pattern matches that of Mountpoint's defaults (even though the objects in S3 may be different from those uploaded by default).

Reply:
I intentionally didn't do that because I am trying to preserve the existing behvaiour with this change, as it's intended to only be a refactor.  Therefore, I'd prefer to leave as-is, but let's discuss if you don't agree.

Reply:
Fair enough, happy to discuss in a later change